Dockerfile

```diff
@@ -26,41 +23,12 @@ RUN python3 -m pip install \
   auditwheel \
   patchelf
 
-RUN gnuArch="$(dpkg-architecture --query DEB_HOST_MULTIARCH)"; \
-ln -s /usr/lib/$gnuArch/libblas.so /usr/lib/$gnuArch/libblas_OPENMP.so
-
 WORKDIR /build
 
-RUN curl -fSsL https://portal.nersc.gov/project/sparse/superlu/superlu_mt_3.1.tar.gz | tar xz \
-  && cd SuperLU_MT_3.1 \
-  && make CFLAGS="-O2 -fPIC -fopenmp" BLASLIB="-lblas" PLAT="_OPENMP" MPLIB="-fopenmp" lib -j1 \
-  && cp -v ./lib/libsuperlu_mt_OPENMP.a /usr/lib \
-  && cp -v ./SRC/*.h /usr/include
-
-RUN git clone --depth 1 -b ${SUNDIALS_VERSION} https://github.com/LLNL/sundials.git \
-  && cd sundials \
-  && mkdir build && cd build \
-  && cmake \
-  -LAH \
-  -DSUPERLUMT_BLAS_LIBRARIES=blas \
-  -DSUPERLUMT_LIBRARIES=blas \
-  -DSUPERLUMT_INCLUDE_DIR=/usr/include \
-  -DSUPERLUMT_LIBRARY=/usr/lib/libsuperlu_mt_OPENMP.a \
-  -DSUPERLUMT_THREAD_TYPE=OpenMP \
-  -DCMAKE_INSTALL_PREFIX=/usr \
-  -DSUPERLUMT_ENABLE=ON \
-  -DSUNDIALS_INDEX_SIZE=32 \
-  -DLAPACK_ENABLE=ON \
-  -DEXAMPLES_ENABLE=OFF \
-  -DEXAMPLES_ENABLE_C=OFF \
-  .. \
-  && make -j4 \
-  && make install
-
 RUN gnuArch="$(dpkg-architecture --query DEB_HOST_MULTIARCH)" \
   && git clone --depth 1 -b Assimulo-3.5.2 https://github.com/modelon-community/Assimulo.git \
   && cd Assimulo \
-  && python3 setup.py bdist_wheel --sundials-home=/usr --blas-home=/usr/lib/${gnuArch} --lapack-home=/usr/lib/${gnuArch} --superlu-home=/usr \
+  && python3 setup.py bdist_wheel \
```

Comment:
We're no longer interested in building a ton of dependencies into the modelica tools as we will require dependencies to be included in the compiled FMU.